lib/u3d/installation.rb

```diff
@@ -179,9 +179,18 @@ def path
 
     def packages
       path = "#{root_path}/Editor/Data/"
-      PlaybackEngineUtils.list_module_configs(path).map do |mpath|
-        PlaybackEngineUtils.module_name(mpath)
+      pack = []
+      PlaybackEngineUtils.list_module_configs(path).each do |mpath|
+        pack << PlaybackEngineUtils.module_name(mpath)
       end
+      pack << 'Documentation' if specific_dir?("#{root_path}/Editor/Data/Documentation/", check_empty: true)
```

Comment:
we could have a method under each PlatformInstallation called

```
def module_files_pattern(module_name)
  return "#{root_path}/Editor/Data/Documentation/*" if module_name == "Documentation"
  return ....
end
```

or something similar

and then do a 
```
['Documentation', 'StandardAssets', 'MonoDevelop'].each do |module_name|
  patern = module_files_pattern(module_name)
  pack << module_name if Dir.glob(pattern).count > 0
end
```

better idea?

Reply:
I installed 2017.3.0f3 on the first slave with all sub-modules, in case you need to find the paths.